scripts/get-version

```diff
@@ -47,11 +47,17 @@ else
     PATCH=0
 fi
 
+# if we're in a features/xxx branch then append `-xxx` to the version as well.
```

Comment:
Do we need this change?

Reply:
I'm a bit worried that this will break other code that parses these versions.

Reply:
nuget doesn't seem to support tags at all. So this is how we can do feature-branches with .net afaict.

Reply:
Can we scope this down to only the .NET version generation, then?

Reply:
sure.  i'll make that change!